projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -1016,8 +1016,13 @@ public function add_subscribers_menu() {
 
 		$blog_id = Connection_Manager::get_site_id( true );
 
+		$source = 'jetpack-menu-calypso-subscribers';
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+			$source = 'jetpack-menu-jetpack-manage-subscribers';
```

Comment:
What's the difference between the following sources?
* `jetpack-menu-calypso-subscribers` v.s `calypso-subscribers`
* `jetpack-menu-jetpack-manage-subscribers` v.s. `jetpack-settings-jetpack-manage-subscribers`

It seems to point to the same page but it's weird that we have different source names 🤔

Reply:
@jeherve might know some history here.

Reply:
I would assume that the 2 sources pointed to a different link at first, and one of them was updated so they now point to the same link.

Reply:
The [Field Guide](https://fieldguide.automattic.com/jetpack-redirects/#naming-and-re-using-sources) encourages multiple sources that point to the same location. `jetpack-menu-calypso-subscribers` and `calypso-subscribers` already existed. I created `jetpack-menu-jetpack-manage-subscribers` and `jetpack-settings-jetpack-manage-subscribers` and the only difference is one source exists for the "jetpack-menu" and the other for "jetpack-settings."